frontend/rust-lib/flowy-database2/src/services/field/type_options/type_option.rs

```diff
@@ -24,6 +24,7 @@ use collab_database::fields::translate_type_option::TranslateTypeOption;
 use collab_database::fields::url_type_option::URLTypeOption;
 use collab_database::fields::TypeOptionData;
 use collab_database::rows::Cell;
+pub use collab_database::template::util::TypeOptionCellData;
```

Comment:
Why do we have a re-export instead of other modules importing from collab?

Reply:
then we don't need to fix all related imports